entrypoint.py

```diff
@@ -116,13 +124,14 @@ def main():
         adapter=os.getenv("INPUT_WAREHOUSE-TYPE"),
         profiles_yml=os.getenv("INPUT_PROFILES-YML"),
         edr_command=os.getenv("INPUT_EDR-COMMAND"),
+        profile_target=os.getenv("INPUT_PROFILE-TARGET") or None,
```

Comment:
We need to add `profile_target` to the `Args` class in order for it to work

Reply:
Good catch, added 👍🏻